src/rules/prefer_english_operator.coffee

```diff
@@ -5,6 +5,7 @@ module.exports = class PreferEnglishOperator
         level: 'ignore'
         message: 'Don\'t use &&, ||, ==, !=, or !'
         doubleNotLevel: 'ignore'
+        ops: ['and', 'or', 'not', 'is', 'isnt']
```

Comment:
This was just a preference thing. Rather than making a new custom rule that does a subset of what this rule does, the rule is now configurable so people can specify *which* english operators they prefer. Much nicer this way, for those who aren't a fan of "isnt" as an operator...